lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -148,35 +163,39 @@ def _get_attached_enodeb_tacs():
     return attached_enodeb_tacs
 
 
-def _get_apn_correction_map_list():
-    mme_config = load_service_mconfig("mme", MME())
-    if len(mme_config.apn_correction_map_list) == 0:
+def _get_apn_correction_map_list(service_config):
+    if len(service_config.apn_correction_map_list) == 0:
```

Comment:
Flip this conditional (make the if statement a `!= 0` -- reads better that way since it groups the usages of the mconfig `service_config.apn_correction_map_list`